### ISSUE TITLE
Add own image + icon size

### ISSUE THREAD

**Issue by Peter8200** [freeotp/freeotp-ios#164]
Dear,

In the latest version for iPhone IOS the option edit is no longer there.  I can't find a possibility to put my own photo/image on an item.  We can't use a predefined icon.

The size of the items is no extremely big.  I should be great if you can choose yourself the size via "preferences". So everybody can do a setup for his own device.

Thanks.

Peter.

**Comment by Peter8200** [comment]
Dear,

A lot of people are missing the possibility to change the icons to easily find back there corresponding application.
For us, we are using free-otp, also for products that are not in the list.
And for one product, we have several items (one for production environment, one for integration one for development).
This is the reason why people like to put there own image to easily find back the right top-code.

Peter.

**Comment by krugerk** [comment]
Agreed. Would be nice to have. I anticipated being able to edit an entry to do this #179, if not initially available during FreeOTP entry creation.

**Comment by jtulak** [comment]
Still no update. :(

I have to move to another app, because while I liked FreeOTP until now, I have dozens of codes and multiple accounts per server - and now the app won't let me differentiate between them easily.

And it even gets the name wrong: e.g. make an entry for a corporate account on Microsoft Office 365 and it will be saved with your company name, not as "Microsoft" like e.g. LastPass authenticator does. So now I have multiple entries (for MS servers, for some company internal servers...) all with the same name and username and absolutely no way to differentiate except an icon which must be picked from a list when creating the entry.

So, unfortunately, I'm forced to another app and I know I won't want to migrate all my accounts back until there is a really big and compelling reason to do all the shuffle.

**Comment by tstimper** [comment]
We need to use customized icons for specific products so it would be good if we either apply our own icons or have our icons in the product. How can we do that?

**Comment by Peter8200** [comment]
I see no evolution.  We have a lot of customers that has chosen a wrong icon.  
It's still a good idea to make this again available so they can customise themselves the app with own pictures/images like before.
It's is very useful in case if you (like us) have a lot of totp items.
The totp link to a specific environment is put in a specific color to make easily the difference between environments.
